### PR TITLE
servicemesh memberroll adjustments + unit test added

### PR DESCRIPTION
**Pull request by elenagerman**
servicemesh memberroll adjustments + unit test added

### CODE REVIEW

File: pkg/servicemesh/memberroll_test.go [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+		if testCase.expectedError != "" {
+			assert.Equal(t, testCase.expectedError, err.Error())
+		} else {
+			assert.Equal(t, testCase.members,
+				testCase.testMemberRoll.Definition.Spec.Members)
+		}
+	}
+}
+
+func TestWithMembers(t *testing.T) {
```
Comment (kononovn) [inline pkg/servicemesh/memberroll_test.go:295]: Please re-name to TestMemberRollWithMembersList

File: pkg/servicemesh/memberroll_test.go [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+		assert.NotNil(t, testMemberRollBuilder.Definition)
+
+		if testCase.expectedError == "" {
+			assert.Equal(t, testCase.name, testMemberRollBuilder.Definition.Name)
+			assert.Equal(t, testCase.namespace, testMemberRollBuilder.Definition.Namespace)
+		}
+	}
+}
+
+func TestMemberRollExist(t *testing.T) {
```
Comment (kononovn) [inline pkg/servicemesh/memberroll_test.go:147]: Please re-name to
TestMemberRollExists

File: pkg/servicemesh/memberroll_test.go [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+		if testCase.expectedError != "" {
+			assert.Equal(t, testCase.expectedError, err.Error())
+		} else {
+			assert.Equal(t, testCase.members,
+				testCase.testMemberRoll.Definition.Spec.Members)
+		}
+	}
+}
+
+func TestMemberRollWithMembers(t *testing.T) {
```
Comment (kononovn) [inline pkg/servicemesh/memberroll_test.go:295]: This function tests > WithMembersList. Can you re-name it to TestMemberRollWithMembersList

File: pkg/servicemesh/memberroll_test.go [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+			if testCase.expectedErrorText != "" {
+				assert.Equal(t, testCase.expectedErrorText, result.errorMsg)
+			}
+		} else {
+			assert.NotNil(t, result)
+			assert.Equal(t, testCase.testMembers, result.Definition.Spec.Members)
+		}
+	}
+}
+
```
Comment (kononovn) [inline pkg/servicemesh/memberroll_test.go:328]: Please add TestMemberRollIsReady test case.

**Review by kononovn — CHANGES_REQUESTED** [review]
Please check few comments above.

**Comment by kononovn** [comment]
@elenagerman Please re-base in order to solve unit test CI issue.

File: pkg/servicemesh/memberroll_test.go (hunk 4 above)
Comment (elenagerman) [reply pkg/servicemesh/memberroll_test.go:328]: how can I test the status of the dummy object?

Comment (kononovn) [reply pkg/servicemesh/memberroll_test.go:328]: Please refer to https://github.com/openshift-kni/eco-goinfra/blob/c6f7a7e7c45e777f1ecf316b87c02ac613ae03d3/pkg/bmh/baremetalhost_test.go#L832

File: pkg/servicemesh/memberroll_test.go (hunk 3 above)
Comment (kononovn) [reply pkg/servicemesh/memberroll_test.go:295]: Can you please re-name function?

**Comment by elenagerman** [comment]
> @elenagerman Please re-base in order to solve unit test CI issue.

rebased yesterday

**Pull request merged**